mass_mailing_resend/readme/CREDITS.md

```diff
@@ -0,0 +1,2 @@
+The migration of this module from 15.0 to 16.0 was financially supported
```

Comment:
Please remove this file, as this doesn't apply for this version.

Reply:
done.